geth/jail/execution_policy.go

```diff
@@ -13,6 +13,43 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/robertkrimen/otto"
 	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/params"
+)
+
+// map of command routes
+var (
+	rpcLocalCommandRoute = map[string]bool{
```

Comment:
It's not exactly what I meant but I also think it's good enough to be merged. 

I meant to remove the map completely and rely on a function (functions) that maps a command name to the execution strategy. Here, if a new Whisper command is added to the protocol, we need to change the code and release a new version of the app. 

This PR is in the queue for a long time so if you could just add a comment with `TODO`, it would be sufficient.

Reply:
Hmmm, I can quick run down up, but that means we would have a map of commands to functions. We can shift it to another issue then.

Reply:
It depends. So, for instance, Whisper commands could be detected with a function:

```go
func (cmd string) (Route, bool) {
  if strings.HasPrefix("shh_", cmd) {
    return LocalRoute, true
  }

  return UnknownRoute, false
}
```

For specific `eth_` commands you would need to still rely on a map but encapsulated within a function with the same signature:
```go
func (cmd string) (Route, bool) {
  return localEthRoutes[cmd]
}
```

Finally, if there is no match, execute remotely.